Restaurants/src/main/java/com/restaurants/entities/RestaurantMenu.java

```diff
@@ -0,0 +1,34 @@
+package com.restaurants.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Data
+@NoArgsConstructor
+@Entity
+public class RestaurantMenu {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long itemId;
+
+    private String itemName;
+
+    private Double price;
+
+    private String description;
+
+    private Boolean vegNonVeg;
+
+    private Long categoryId;
+
+    private Long restaurantId;
+
+    private String imageUrl;
```

Comment:
check the datatype and use appropriate datatype

Reply:
Okay